src/catalog/src/builtin.rs

```diff
@@ -2862,15 +2862,51 @@ LEFT JOIN mz_catalog.mz_databases d ON d.id = s.database_id
 WHERE s.database_id IS NULL OR d.name = current_database()",
 };
 
+// Note: Databases, Roles, Clusters, Cluster Replicas, Secrets, and Connections are excluded from
+// this view.
 pub const PG_DESCRIPTION: BuiltinView = BuiltinView {
     name: "pg_description",
     schema: PG_CATALOG_SCHEMA,
-    sql: "CREATE VIEW pg_catalog.pg_description AS SELECT
-    c.oid as objoid,
-    NULL::pg_catalog.oid as classoid,
-    0::pg_catalog.int4 as objsubid,
-    NULL::pg_catalog.text as description
-FROM pg_catalog.pg_class c",
+    sql: "CREATE VIEW pg_catalog.pg_description AS
+SELECT
+	pg_obj.objoid,
+	pg_obj.classoid,
+	COALESCE(cmt.object_sub_id::INT4, 0) AS objsubid,
```

Comment:
```suggestion
	COALESCE(cmt.object_sub_id::pg_catalog.int4, 0) AS objsubid,
```

Reply:
This cast would fail if we had more than i64::max columns when casting from the underlying u64. So certainly is fine, but annoying we even have this idea mismatch in the first place. Should we make/have made sub_id an i64?

Reply:
I'm not opposed to changing `mz_comments` to an i64. I believe we populate the table with `usize`s in Rust which is probably why `uint8` was chosen. CC @ParkMyCar

Reply:
Ahhh yeah I used a `uint8` exactly because internally the type is a `usize`. I added some TODOs and filed https://github.com/MaterializeInc/materialize/issues/22246 to refactor this to be cleaner.